golden/clr-angular.d.ts

```diff
@@ -943,6 +1045,37 @@ export declare class ClrPopoverToggleService {
     toggleWithEvent(event: any): void;
 }
 
+export declare class ClrProgressBar {
+    readonly ariaLive: "assertive" | "off" | "polite";
+    assertive: boolean;
+    clrDanger: boolean | string;
+    clrFade: boolean | string;
+    clrFlash: boolean | string;
+    clrFlashDanger: boolean | string;
+    clrLabeled: boolean | string;
+    clrLoop: boolean | string;
+    clrSuccess: boolean | string;
+    readonly dangerClass: boolean;
+    readonly displayValue: string;
+    displayval: string;
```

Comment:
Shouldn't this be camelCase? 
Totally unrelated to popovers, I think, but it appears in your commit 1st :)

Reply:
Talked with Bozhidar, this is due to exposing native API. Still unrelated to your PR :)